src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -221,21 +253,34 @@ public Delegate[] CreateGetters(IRow input, Func<int, bool> activeOutput, out Ac
 
             public RowMapperColumnInfo[] GetOutputColumns()
             {
-                return new[] { new RowMapperColumnInfo(_outputColName, _outputColType, null) };
+                var info = new RowMapperColumnInfo[_outputColNames.Length];
+                for (int i = 0; i < _outputColNames.Length; i++)
+                {
+                    info[i] = new RowMapperColumnInfo(_outputColNames[i], _outputColTypes[i], null);
+                }
+                return info;
             }
 
-            private static (ColumnType, TFDataType) GetOutputTypes(TFGraph graph, string columnName)
```

Comment:
>static [](start = 20, length = 6)

Can this still be static?

Reply:
I am using the ` _session.Graph[name]` in this method.

---
In reply to: [215388432](https://github.com/dotnet/machinelearning/pull/814#discussion_r215388432) [](ancestors = 215388432)

Reply:
You can use the graph that is passed as an argument instead.

---
In reply to: [215390484](https://github.com/dotnet/machinelearning/pull/814#discussion_r215390484) [](ancestors = 215390484,215388432)